OpenRA.Mods.Common/Traits/PaletteEffects/RotationPaletteEffect.cs

```diff
@@ -106,11 +106,12 @@ public void AdjustPalette(IReadOnlyDictionary<string, MutablePalette> palettes)
 			}
 		}
 
-		static bool AnyPaletteNameStartsWith(HashSet<string> names, string prefix)
+		static bool AnyPaletteNameStartsWith(HashSet<string> names, string currentPalette)
```

Comment:
I'd change this to something like `static bool startsWithAny(string name, HashSet<string> prefixes)`.